frontend/src/metabase/core/components/Select/Select.jsx

```diff
@@ -268,6 +271,7 @@ class Select extends Component {
           searchPlaceholder={searchPlaceholder}
           hideEmptySectionsInSearch={hideEmptySectionsInSearch}
         />
+        {footerFn && footerFn()}
```

Comment:
you could use optional chaining syntax for this. That make it a bit more concise.
```suggestion
        {footerFn?.()}
```